vspace/src/main/java/edu/asu/diging/vspace/core/services/impl/ExhibitionManager.java

```diff
@@ -66,4 +81,79 @@ public IExhibition getStartExhibition() {
         }
         return null;
     }
+
+    /**
+     * Updates the Exhibition with given list of languages. It fetches the language from exhibitionLanguageConfig using code.
+     *  
+     * @param exhibition
+     * @param defaultLanguage 
+     * @param languages
+     * @throws LanguageListConfigurationNotFound 
+     */
+    @Override
+    public void updateExhibitionLanguages(Exhibition exhibition, List<String> codes, String defaultLanguage) throws LanguageListConfigurationNotFound {
+        if(CollectionUtils.isNotEmpty(codes)) {
+            
+            // Adds defaultLanguage to codes list if not already exists.
+            if(defaultLanguage!=null && !codes.contains(defaultLanguage)) {
+                codes.add(defaultLanguage);
+            }
+
+
+            if(CollectionUtils.isNotEmpty(exhibitionLanguageConfig.getExhibitionLanguageList())) {
```

Comment:
check this right at the beginning. no need to do anything if there is no exhibition language list.